### DISCUSSION TITLE
The Extra Add-on

### DISCUSSION THREAD

**Discussion by ReparkHjc** [DLR-VSDC/IEEE-MVC-2023#42] [Q&A]
Is it possible to include information about the Add-on components required for our solution, which are Add-ons related to deep learning and statistical machine learning, in the ReadMe file and prompt their installation during the evaluation process?

**Comment by imeb** [comment]
Dear @ReparkHjc,
As long as the toolbox or add-ons are accessible to us, and you have a clear description of the minimum requirements for us to know and install, I assume it is a feasible solution. Please remember to test them on MATLAB R2021B.
Best,
Iman
